app/js/actions/connection.js

```diff
@@ -1,10 +1,7 @@
-import fetch from 'isomorphic-fetch';
-import es6Promise from 'es6-promise';
 import CryptoJS from 'crypto-js';
 
 import actions from './index';
 
-es6Promise.polyfill();
```

Comment:
Are these (isomorphic-fetch, es6-promise) being scrapped because we aren't needing cross-browser support anymore? I'll have to do some digging, because postcss' autoprefixer might also be fine to scrap now.